blacklisted-modules.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Since we're importing this into a node script, perhaps we can turn this into a js file, which will allow us to leave comments and the like :).

It might also be worth adding a bit of documentation around this, e.g. a `docs/CONFIGURATION.md`